eos_data_distribution/parallel.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, workers, type="Batch"):
         self._incomplete_workers = set(workers)
         for batch in self._incomplete_workers:
             batch.connect('complete', self._on_batch_complete)
+            worker.start()
```

Comment:
Should rename this to `for worker in self._incomplete_workers`, or this should be `batch.start()`